web/server/vue-cli/src/views/NewFeatures.vue

```diff
@@ -2,6 +2,296 @@
   <!-- eslint-disable max-len -->
   <v-container fluid>
     <v-timeline align-top>
+      <v-timeline-item fill-dot icon="mdi-star" color="green lighten-1">
+        <new-release-item color="green lighten-1">
+          <template v-slot:title>
+            <a
+              href="https://github.com/Ericsson/codechecker/releases/tag/v6.17.0"
+              target="_blank"
+              class="white--text"
+            >
+              Highlights of CodeChecker 6.17.0 release
+            </a>
+          </template>
+
+          <new-feature-item>
+            <template v-slot:title>
+              Git blame integration
+            </template>
+
+            <v-row>
+              <v-col cols="8">
+                With this feature it will be possible for a developer to check
+                who modified the source line last where a CodeChecker error
+                appears.
+
+                <ul>
+                  <li>
+                    If the project which was analyzed is a git repository
+                    CodeChecker store command will store blame information for
```

Comment:
```suggestion
                    <tt>CodeChecker</tt> store command will store blame information for
```
We should emphasize these commands to distinguish them from normal text. It occurs several places in this patch.

Reply:
Okay, I used the code tag to highlight these code snippets.